Gulpfile.mjs

```diff
@@ -283,6 +283,16 @@ function buildRollup(packages, targetBrowsers) {
             BABEL_VERSION: JSON.stringify(babelVersion),
             VERSION: JSON.stringify(version),
           }),
+          rollupCommonJs({
+            include: [
+              /node_modules/,
+              "packages/babel-runtime/regenerator/**",
+              "packages/babel-preset-env/data/*.js",
+              // Rollup doesn't read export maps, so it loads the cjs fallback
+              "packages/babel-compat-data/*.js",
+              "packages/*/src/**/*.cjs",
+            ],
+          }),
```

Comment:
Q: Does the plugin reordering here resolve https://github.com/rollup/plugins/issues/557?

Reply:
Yes, I don't get any error now. Btw, I swapped them because it's documented at https://github.com/rollup/plugins/tree/master/packages/babel#using-with-rollupplugin-commonjs

This means that we cannot use flow/TS in cjs files, but I couldn't find another way to make it work.